### PR TITLE
[zh] Sync Ambient new gat stard traffic manage and clean into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Sync Ambient new gat stard traffic manage and clean into Chinese

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/en/docs/ambient/getting-started/cleanup/index.md
Comment (windsonsea) [inline content/en/docs/ambient/getting-started/cleanup/index.md:2]: oh, this en file shall be removed from your commit before zh-approvers to merge this.

Comment (wilsonwu) [reply content/en/docs/ambient/getting-started/cleanup/index.md:2]: Sure, let's fix en typo in other PR.

**Review by windsonsea — APPROVED** [review]
thanks

**Pull request merged**
